packages/admin-panel/src/table/StyledReactTable.jsx

```diff
@@ -32,11 +27,11 @@ export const StyledReactTable = styled(ReactTable)`
 
     .rt-th {
       font-weight: 500;
-      font-size: 16px;
-      line-height: 19px;
+      font-size: 1rem;
       color: ${COLORS.TEXT_DARKGREY};
       text-align: left;
-      padding: 6px 5px;
+      padding-block: 0.375rem;
+      padding-inline: 0.32rem;
```

Comment:
This will probably get refactored out in the redesign, but just in case:

```suggestion
      padding: 0.375rem;
```

Don’t think the 0.88px difference is doing much 😆